env.example

```diff
@@ -3,3 +3,9 @@ NODE_ENV=development
 
 # PORT is the port used by the web server
 PORT=8080
+
+# LOG_LEVEL is used to set the level of debugging for the logs.
```

Comment:
Also mention `info` and `error` and add a link to this so people can look up the options http://getpino.io/#/docs/api?id=level

Reply:
Done.